src/devices/vdml_distance.cpp

```diff
@@ -48,5 +48,10 @@ std::ostream& operator<<(std::ostream& os, pros::Distance& distance) {
     return os;
 }
 
+namespace literals {
+const pros::Distance operator"" _dist(const unsigned long long int d) {
+    return pros::Distance(d);
+}
+} // namespace literals
 }
 }  // namespace pros
```

Comment:
mind adding a newline at the end of this file? Not your thing but it doesn't seem to exist on this file for some reason.

Reply:
I added it